2017/solutions/kaib/DA/requirements.txt

```diff
@@ -0,0 +1 @@
+tensorflow_gpu==0.13.0
```

Comment:
it seems to me, there are some dependencies missing, like numpy and nltk

Reply:
What makes you think that some dependencies are missing?

Reply:
because you're using additional software like NLTK, so you should add it as perendency explicitly

Reply:
We updated readme & requirements.txt, which lists all required external libraries.